modules/monitoring-modifying-retention-time-and-size-for-prometheus-metrics-data.adoc

```diff
@@ -37,113 +52,73 @@ If any data blocks exceed the defined retention time or the defined size limit,
 ====
 Data compaction occurs every two hours. Therefore, a persistent volume (PV) might fill up before compaction, potentially exceeding the `retentionSize` limit. In such cases, the `KubePersistentVolumeFillingUp` alert fires until the space on a PV is lower than the `retentionSize` limit.
 ====
+// The section above will be removed and made into its separate concept module.
 
 .Prerequisites
 
+// tag::CPM[]
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+* You have created the `cluster-monitoring-config` `ConfigMap` object.
```

Comment:
Add attributes here for the attributes defined at the start of this file?

Reply:
No need here, because the prerequisite section is divided by tags 👍